app/uk/gov/hmrc/incometaxsessiondata/controllers/SessionController.scala

```diff
@@ -35,7 +36,7 @@ class SessionController @Inject()(cc: ControllerComponents,
     extends BackendController(cc) with Logging {
 
   def getById(sessionID: String): Action[AnyContent] = authentication.async {  _ =>
-    sessionService.get(sessionID) map {
+    sessionService.get(SessionId(sessionID)) map {
```

Comment:
I was expecting change in method signature like:
`def getById(sessionID: SessionId)`